### DISCUSSION TITLE
Cron without PHP CLI

### DISCUSSION THREAD

**Discussion by robertkrzuk** [phalcon/cphalcon#16572] [Q&A]
Hello,
I have got problem with my hosting provider. I have got access to PHP8.x with Phalcon 5.6 however I don't have access to PHP8.x CLI with Phalcon enabled. Because of that I'm unable to run CRON Tasks from CLI. Is there a possibility to run Phalcon Cron Task with http (CURL) or maybe you have some idea how to manage that?

**Comment by s-ohnishi** [comment]
> I don't have access to PHP8.x CLI with Phalcon enabled

I rent a VPS and can use Phalcon from the web server and the mail server (sorry, I don't use cron).
Depending on the server OS, you can specify whether or not to use each SAPI.
So it may be the fault of the server you rent, or it may be the fault of your missing settings.

If you can't use it from cron, you can use curl or something similar to access it from another server, but wouldn't it be more beneficial to move to a server where you can do what you want, rather than using a different server just for that?
